SourceryTests/Stub/JavaScriptTemplates/Includes.js

```diff
@@ -0,0 +1 @@
+<%- include('Equality.js') %>-%><%- include('Other.js') %>-%>
```

Comment:
Ok, I didn't really know what you meant – I never used the `-%>` with a starting tag. If you want I can change it to `<% -%>`. I'll look into the file extension requirement after we discussed if the file extension should be changed to `ejs` or not